full-service/src/service/wallet_impl.rs

```diff
@@ -57,39 +58,43 @@ pub struct WalletService<
     T: BlockchainConnection + UserTxConnection + 'static,
     FPR: FogPubkeyResolver + Send + Sync + 'static,
 > {
-    wallet_db: WalletDb,
+    pub wallet_db: WalletDb,
```

Comment:
Whats accessing this externally?

Reply:
This unfortunately was a requirement when I refactored out the PasswordManager service to its own trait. Maybe I'm missing some rust trick so that I can access the private members of the WalletService while implementing the PasswordManager trait for it?

Reply:
Can you point me at where this is being used in the trait implementation?

Reply:
Here's an example: https://github.com/mobilecoinofficial/full-service/blob/880f94b73ffd8c4fb846430d1011367cc8758c76/full-service/src/service/password_manager.rs#L121

Reply:
Ha, I'm surprised that is the case. I guess Rust considers it a separate module if its in a separate file.